src/main/java/uk/gov/defra/datareturns/validation/activities/ActivityValidator.java

```diff
@@ -79,17 +82,30 @@ private boolean checkUniqueRiverPerSubmission(final Activity activity, final Con
     }
 
     /**
-     * Check that the count of days spent on the river is within possible maximums
+     * Check that the count of days spent on the river is within possible maximums.
+     * Zero is permitted for the FMT user (Paper submissions)
      *
      * @param activity the activity to be validated
      * @param context  the validator context
      * @return true if valid, false otherwise
      */
     private boolean checkDays(final Activity activity, final ConstraintValidatorContext context) {
         boolean valid = checkDaysMandatoryRelease(activity, context) && checkDaysOther(activity, context);
-        if (valid && activity.getDaysFishedWithMandatoryRelease() < 1 && activity.getDaysFishedOther() < 1) {
-            valid = handleError(context, "DAYS_FISHED_NOT_GREATER_THAN_ZERO", ConstraintValidatorContext.ConstraintViolationBuilder::addBeanNode);
+
+        final Optional<Submission> sub = Optional.ofNullable(activity.getSubmission());
+
+        if (sub.isPresent()) {
+            if (valid && sub.get().getSource().equals(SubmissionSource.WEB)
+                    && activity.getDaysFishedWithMandatoryRelease() < 1 && activity.getDaysFishedOther() < 1) {
+                valid = handleError(context, "DAYS_FISHED_NOT_GREATER_THAN_ZERO", ConstraintValidatorContext.ConstraintViolationBuilder::addBeanNode);
+            }
+
+            if (valid && sub.get().getSource().equals(SubmissionSource.PAPER)
+                    && activity.getDaysFishedWithMandatoryRelease() < 0 && activity.getDaysFishedOther() < 0) {
+                valid = handleError(context, "DAYS_FISHED_LESS_THAN_ZERO", ConstraintValidatorContext.ConstraintViolationBuilder::addBeanNode);
+            }
```

Comment:
Need to add a test which checks that an appropriate error is generated for paper returns with negative effort values.

Is there a corresponding PR on the frontend for the change of validation message?

Reply:
The front end will just show "Enter a number between 0 and 167/198" if you add a negative effort 
The 0 is allowed in one box but not both for anglers of both for FMT.